transcriptic/cli.py

```diff
@@ -484,6 +484,46 @@ def launch_cmd(ctx, protocol, project, save_input, local, accept_quote, params,
     api = ctx.obj.api
     commands.launch(api, protocol, project, save_input, local, accept_quote, params, pm=None, test=None, pkg=None)
 
+@cli.command('preview-params')
+@click.argument('protocol')
+@click.option(
+    '--project', '-p',
+    metavar='PROJECT_ID',
+    required=True,
```

Comment:
this should be not required, if we're allowing local launches, if we're not, then we should change the logic below to reflect this